test/testcafe/spec/EnrollAuthenticatorEmail_spec.js

```diff
@@ -330,28 +339,29 @@ test
     await t.expect(firstRequestUrl).eql('http://localhost:3000/idp/idx/challenge/poll');
 
     jsonBody = JSON.parse(lastRequestBody);
-    await t.expect(jsonBody).eql({'stateHandle':'eyJ6aXAiOiJER'});
+    // in V3, it's { resend: true, stateHandle: 'eyJ6aXAiOiJER' }
+    await t.expect(jsonBody).contains({'stateHandle':'eyJ6aXAiOiJER'});
```

Comment:
I think this is a reasonable change that doesn't necessarily require a comment to explain it

Reply:
No harm in keeping the comment I guess.